home.css

```diff
@@ -0,0 +1,46 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    background: #191927;
```

Comment:
you can clearly see the color difference between the background of the image and the other color

![Screenshot from 2021-04-13 23-00-42](https://user-images.githubusercontent.com/47992412/114613588-3a382c80-9cac-11eb-9188-3f680bdbfedd.png)

use `#211c2c`
```suggestion
    background: #211c2c;
```